engine/gui/src/gui_script.cpp

```diff
@@ -1313,6 +1314,10 @@ namespace dmGui
             Vector4 original = dmGui::GetNodePropertyHash(scene, hnode, property_hash);
             to = Vector4(*v3, original.getW());
         }
+        else if ((q = dmScript::ToQuat(L, 3)))
+        {
+            to = Vector4(*q);
+        }
```

Comment:
While the end result isn't what you'd expect (lerp'ing a quaternion, which is another issue), it should still be legal to pass the output from `gui.get_rotation()`to `gui.animate("rotation")`